style.css

```diff
@@ -0,0 +1,222 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.container {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.bg-black {
+  background-color: black;
+}
+
+.text-white {
+  color: white;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  height: 534px;
+  background-image: url("../images/your_verse_hero.jpg");
+  background-size: 1440px 1315px;
+  background-position-y: 534px;
+  text-align: center;
+}
+
+nav ul li a {
+  color: white;
+}
+
+.header-text a {
+  color: white;
+  font-size: 25px;
+  text-decoration: none;
+}
+
+#main-nav {
+  display: flex;
+  align-items: flex-end;
+  width: 980px;
+  height: 114px;
+}
+
```

Comment:
Why are you making this width to be 980px? Consider using a value in `%` here so it adjusts to different screen sizes.